common/djangoapps/microsite_configuration/backends/filebased.py

```diff
@@ -0,0 +1,199 @@
+"""
+Microsite backend that reads the configuration from a file
+
+"""
+import edxmako
+import os
+import threading
+
+from django.conf import settings
+from microsite_configuration.backends.base import AbstractBaseMicrositeBackend
+
+
+class SettingsFileMicrositeBackend(AbstractBaseMicrositeBackend):
```

Comment:
Or if we want to go with SettingsFileMicrositeBackend that is fine, but we should rename the module to settings_file.py